openapi/components/schemas/Order.yaml

```
@@ -0,0 +1,159 @@
+type: object
+required:
+  - customerId
+  - websiteId
+  - items
+properties:
+  id:
+    type: string
+    description: ID of the order.
+    readOnly: true
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  organizationId:
+    readOnly: true
+    allOf:
+      - $ref: ./OrganizationId.yaml
+  customerId:
+    $ref: ./CustomerId.yaml
+  websiteId:
+    $ref: ./WebsiteId.yaml
+  initialInvoiceId:
+    type: string
+    description: ID of the initial invoice.
+    readOnly: true
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  recentInvoiceId:
+    description: |-
+      ID of the most recently issued invoice.
+      The invoice might not be `paid` yet.
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  upcomingInvoiceId:
+    type: string
+    description: ID of the upcoming invoice.
+    readOnly: true
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  billingStatus:
+    description: |-
+      Temporary billing status of the most recent invoice.
+      This value may help you to determine if you should change the service status of the service,
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  items:
+    type: array
+    description: Items of the order.
+    items:
+      type: object
+      description: Order entry details.
+      discriminator:
+        propertyName: type
+        mapping:
+          subscription: ./Subscription.yaml
+          one-time-sale: ./OneTimeSale.yaml
+      anyOf:
+        - $ref: ./Subscription.yaml
+        - $ref: ./OneTimeSale.yaml
+  invoiceIssueTimeShift:
+    description: |-
+      Invoice issue time shift interval in ISO-8601 format.
```

suggestion: to add a definition for invoice time shift
```suggestion
      Invoice issue time shift interval in ISO-8601 format.
      
      Invoice issue time shift adjusts the issue date and time of renewal invoices for this subscription.
```